src/Components/WidgetDrawer/WidgetDrawer.tsx

```diff
@@ -100,6 +102,10 @@ const AddWidgetDrawer = ({ children }: AddWidgetDrawerProps) => {
       </Split>
       <Gallery className="widg-l-gallery pf-v5-u-pt-sm" hasGutter>
         {Object.entries(widgetMapping).map(([type, { config }], i) => {
+          const requiresOrgAdmin = isOrgAdminWidgetPermissionRequired(config);
+          if (requiresOrgAdmin && !currentUser?.is_org_admin) {
+            return null;
```

Comment:
While this prevents the user from adding/viewing this widget from the UI - we may also want to remove the widget in our update API endpoint in the backend for complete coverage (a bit more challenging as we would need to decode the user permissions in chrome-service-backend from the JWT).